library.properties

```diff
@@ -1,8 +1,9 @@
-name=arduino_TLC59116
-version=1.5
-author=Alan Grover
+name=TLC59116
+version=1.8
+author=Red Byer (original by: Alan Grover)
```

Comment:
I'm happy to add you as contribute, second author

Reply:
Change as suits you -- I did this more so that I don't mix up my modifications to your library.  For your branch on GitHub you can even remove my name (I don't merit author status based on my trivial edits).